src/components/pages/CitySearch/Index.js

```diff
@@ -1,31 +1,50 @@
 import React from 'react';
 
 import logic from './logic.js';
+import dummyData from '../../../utils/dummyData';
 // import map from 'map';
 // import
 // import Header from '../../common/header';
 
 logic();
 
-export default () => (
-  <div class="citySearch">
-    {/* nav */}
-    <div>
-      <div>
-        {/* left */}
-        <h2>city</h2>
-        <p>description</p>
-      </div>
-      {/* left */}
-      <div>
-        {/* right */}
-        <div>Map</div>
-        <div>
-          <div>graph</div>
-          <div>data</div>
-        </div>
+export default function CitySearch() {
+  const {
+    citynamestate,
+    wikiimgurl,
+    population,
+    densitymisq,
+    averageage,
+    costoflivingindex,
+    individualincome,
+    householdincome,
+    averagehouse,
+    rent,
+    averagetemp,
+  } = dummyData;
+  return (
+    <div class="citySearch">
+      {/* nav */}
+      <div class="citySearchCont">
+        <section
+          class="citySearchTextDescription"
+          style={{ backgroundImage: `url(${wikiimgurl})` }}
```

Comment:
I'm not fond of inline styles. Maybe remove and place inside of a .less/.css file OR use something like styled-components.

Reply:
The URL I am using is coming from state. Can we add data from our API to a less file somehow?